R/readUNFCCC_NDC.R

```diff
@@ -64,18 +69,134 @@ readUNFCCC_NDC <- function(subtype, subset) {
         select(
           "ISO_Code" = 2, "Reference_Year" = 7,
           "BAU_or_Reference_emissions_in_MtCO2e" = 8, "Target_Year" = 9,
-          "Type" = 10, "LULUCF" = 11, "Unconditional Absolute" = 12, 
-          "Conditional Absolute" = 13, "Unconditional Relative" = 14, 
-          "Conditional Relative" = 15
+          "Type" = 10, "LULUCF" = 11, "Unconditional Relative" = 12, 
+          "Conditional Relative" = 13, "Unconditional Absolute" = 14, 
+          "Conditional Absolute" = 15
         )
     } else {
       input <- input %>%
         select(
           "ISO_Code" = 2, "Reference_Year" = 7,
           "BAU_or_Reference_emissions_in_MtCO2e" = 8, "Target_Year" = 9,
-          "Type" = 10, "Unconditional Absolute" = 11, "Conditional Absolute" = 12,
-          "Unconditional Relative" = 13, "Conditional Relative" = 14
+          "Type" = 10, "Unconditional Relative" = 11, "Conditional Relative" = 12,
+          "Unconditional Absolute" = 13, "Conditional Absolute" = 14
+        )
+    }
+    
+    
+    if (any(grepl("2025", subtype, fixed = TRUE))) {
+      input2035 <- readxl::read_excel(
+        NDC2035, sheet = "NDC overview", skip = 1, na = c("?", ""), progress = FALSE
+      ) %>%
+        suppressMessages() %>%
+        filter(!is.na(`Gas coverage`)) %>%
+        rename(
+          ISO_Code = ISO3,
```

Comment:
write "ISO_Code" = .data$ISO3 to please the linter. same below. find all errors by running linting on the file. also apply auto format.

Reply:
Sorry, correction in `dplyr::rename`, the correct rewrite should be `ISO_CODE" = "ISO3"